### PR TITLE
xenos can hold xeno toys

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Xenos can now hold the xenomorph action figure

## Why It's Good For The Game

I think it's a minor interaction and xenos don't really have much to hold except for facehuggers.

## Changelog

:cl:
qol: Xenomorphs can now pick up xenomorph action figures.
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -952,6 +952,7 @@
 	name = "xenomorph action figure"
 	desc = "MEGA presents the new Xenos Isolated action figure! Comes complete with realistic sounds! Pull back string to use."
 	w_class = WEIGHT_CLASS_SMALL
+	item_flags = XENOMORPH_HOLDABLE
```
Comment (Zonespace27) [inline code/game/objects/items/toys.dm:955]: They should be able to hold the `/obj/item/toy/plush/rouny` too

Comment (JohnFulpWillard) [reply code/game/objects/items/toys.dm:955]: wtf is a rouny

Comment (private-tristan) [reply code/game/objects/items/toys.dm:955]: image

Comment (Zonespace27) [reply code/game/objects/items/toys.dm:955]: image

Red beno

Comment (Jackal-boop) [reply code/game/objects/items/toys.dm:955]: rouny :)

**Comment by improvedname** [comment]
let xeno hold ralsei aswell 👿

**Comment by Toastgoats** [comment]
Knowing and loving rouny should be a requirement for making xeno prs

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (Toastgoats) [reply code/game/objects/items/toys.dm:955]: John you're literally a maintainer how did you not know about rouny he's straight up tgmc's mascot at this point

Comment (JohnFulpWillard) [reply code/game/objects/items/toys.dm:955]: what's a tgmc

**Pull request merged**